nuxt.config.ts

```diff
@@ -1,4 +1,6 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
-
-})
+  typescript: {
+    typeCheck: true,
+  },
+});
```

Comment:
can you also add `dev: true,
    debug: true,` properties based on .env? without that vue dev tool do not work

Reply:
sure :yes_sir: